apps/penxle.com/src/lib/tiptap/components/TiptapRenderer.svelte

```diff
@@ -38,8 +37,10 @@
   });
 </script>
 
-<div bind:this={element} class={clsx(...(editor ? [] : ['ProseMirror', $$props.class]))} {...$$restProps}>
+<div bind:this={element} class={$$props.class} {...$$restProps}>
```

Comment:
<img width="358" alt="스크린샷 2023-11-14 오후 10 38 27" src="https://github.com/penxle/penxle/assets/5278201/73253d23-32c9-4eec-9be3-6303359dc561">

P2: tiptap Editor 인스턴스 때문에 클래스가 중복으로 들어가고 있습니다. (`bodylong-16-m`)

Tiptap Editor props 중에 editorProps 필드를 지우고 아래처럼 수정하는게 어떨까요?

```suggestion
<div bind:this={element} {...$$restProps}>
```